src/vast_tracker.js

```diff
@@ -612,13 +612,12 @@ export class VASTTracker extends EventEmitter {
     }
     if (
       this.creative &&
-      this.creative.universalAdId &&
-      this.creative.universalAdId.idRegistry &&
-      this.creative.universalAdId.value
+      this.creative.universalAdIds &&
+      this.creative.universalAdIds.length
```

Comment:
What do you think of using the optional chaining operator for l.617 to 616 ? 
```suggestion
      this.creative?.universalAdIds?.length
```

Reply:
Updated